### DISCUSSION TITLE
NVIDIA DeepStream SDK equivalent

### DISCUSSION THREAD

**Discussion by jstumpin** [ROCm/ROCm#4059] [Q&A] [answered]
When will AMD **GPU** invest into such a stack? AMD FPGA has already [implemented it years ago](https://www.xilinx.com/developer/articles/vvas-migration-for-deepstream-users.html). AMD is losing competitive ground each passing year without **GStreamer integration**.

**Comment by tcgu-amd** [comment, marked as answer]
Hi @jstumpin, thank you for your feedback! We are aware of the lack of DeepStream SDK in ROCm stack at the moment. We will continue to try our best to bring as much feature to ROCm as possible, so please keep an eye out for official announcements in future ROCm releases. Thanks!
